website/data/docs-nav-data.json

```diff
@@ -375,8 +375,17 @@
     ]
   },
   {
-    "title": "Consul API Gateway <sup>BETA</sup>",
-    "path": "api-gateway"
+    "title": "Consul API Gateway",
+    "routes": [
+      {
+        "title": "Overview",
+        "path": "api-gateway"
```

Comment:
I adjusted the nav and file structure in our directory to follow this convention from most of the other sections. This way, we don't break the link for the docs that are already out there today. If someone strongly doesn't like "Overview" as the page title in the nav bar, then we can change it.

Reply:
I was going to do exactly that but you beat me to it.